app/serializers/api/v3/serializable_provider_suggestion.rb

```diff
@@ -0,0 +1,9 @@
+module API
+  module V3
+    class SerializableProviderSuggestion < JSONAPI::Serializable::Resource
+      type "provider"
+
+      attributes :provider_name, :provider_code
```

Comment:
Do we need One More Serializer (:tm:) here? We should be able to just use `SerializableProvider` with sparse fields?

Reply:
Yeah sure - I think as long as there are not plans to add many more attributes?

Reply:
If we need to make a new serialiser, we always can ... until then I think we should try to stick to one serializer per resource / API version combination.